lib/loader.js

```diff
@@ -20,8 +20,17 @@ module.exports = function(content, map) {
 	var resolve = createResolver(query.alias);
 
 	if(sourceMap) {
-		if (map && typeof map !== "string") {
-			map = JSON.stringify(map);
+		if (map) {
+			if (typeof map === "string") {
+				map = JSON.stringify(map);
+			}
+
+			if (map.sources) {
+				map.sources = map.sources.map(function (source) {
+					return source.replace(/\\/g, '/');
```

Comment:
@michael-ciniawsky This is a source map check. Can we assume forward slash there due to browser environment?